core/jazz_cloud-logs-streamer/index.js

```diff
@@ -168,7 +167,7 @@ function transform(payload) {
                         data.message = message;
                     }
                     data.log_level = utils.getSubInfo(logEvent.message, globalConfig.PATTERNS.log_level, 0);
-                    if (_.isEmpty(data.log_level)) {
+                    if (isEmpty(data.log_level)) {
```

Comment:
Instead of isEmpty function, you can write if(!data.log_level).

Reply:
isEmpty function has been removed